R/MeasureSurvAUC.R

```diff
@@ -48,15 +48,15 @@ MeasureSurvAUC = R6Class("MeasureSurvAUC",
       }
 
       args$times = ps$times
-      if (length(args$times) == 0) {
-        args$times = sort(unique(prediction$truth[, 1]))
+      if (length(args$times) == 0L) {
+        args$times = sort(unique(prediction$truth[, 1L]))
       }
 
       if ("Surv.rsp.new" %in% names(formals(FUN))) {
         args$Surv.rsp.new = prediction$truth # nolint
       }
 
-      auc = mlr3misc::invoke(FUN, lpnew = prediction$lp, .args = args)
```

Comment:
also this one - how can someone just take back this changed line?

Reply:
I can always revert those changes in case you don't want to deal with merge conflicts later